update/application.go

```diff
@@ -336,6 +351,55 @@ func deleteWorkerJob(name string, cfg *apps.Config) {
 	cfg.WorkerJobs = newJobs
 }
 
+func (job scheduledJob) applyUpdates(cfg *apps.Config) {
+	if (job.Command != nil || job.Size != nil) && job.Name == nil {
```

Comment:
I think a `job.Name` is needed all the time, right? So a first check for

```
if job.Name == nil {
  return
}
```

should be done.